src/backends/pgfplots.jl

```diff
@@ -98,6 +98,12 @@ const _pgf_series_extrastyle = KW(
     :xsticks => "xcomb",
 )
 
+const _pgf_annotation_halign = KW(
+    :center => "",
+    :left => "right",
```

Comment:
Doesn't matter much but for something like this I would have put in a short comment.